### ISSUE TITLE
[Suggestion] Open keyboard on first tap

### ISSUE THREAD

**Issue by Stanzilla** [android-police/apkmirror-public#2]
#### Expected behavior
On mobile, when you tap the search icon the keyboard should pop up

#### Actual behavior
You have to tap the search icon and then into the search bar to pop up the keyboard

#### Steps to reproduce the problem

Tap the search icon

**Comment by archon810** [comment]
Good suggestion, it's been bothering me too. We'll get it done!

**Comment by archon810** [comment]
This is now live.

**Comment by Stanzilla** [comment]
@archon810 this is broken now for me, can you repro?

**Comment by archon810** [comment]
Yeah, it seems like the fix isn't applied to some pages. It does work on the homepage, for example.

We'll fix it.

**Comment by SeBsZ** [comment]
Fix has been created. Just hang on until it gets pushed live ASAP. Thanks for the report!
